packages/nuxt/src/pages/utils.ts

```diff
@@ -70,8 +70,13 @@ export async function resolvePagesRoutes (pattern: string | string[], nuxt = use
     return pages
   }
 
+  const extraPageMetaExtractionKeys = nuxt.options?.experimental?.extraPageMetaExtractionKeys
+
   const augmentCtx = {
-    extraExtractionKeys: new Set(['middleware', ...nuxt.options.experimental.extraPageMetaExtractionKeys]),
+    extraExtractionKeys: new Set([
+      'middleware',
```

Comment:
@danielroe maybe you know if `defaultExtractionKeys` can be used here instead of just adding `'middleware'`? Came up [here](https://github.com/nuxt/nuxt/pull/32510/files#r2176243781).